.gitignore

```diff
@@ -1 +1,7 @@
+<<<<<<< HEAD
```

Comment:
There are some **merge conflicts** in your project https://snipboard.io/jwkY2r.jpg . You should correct them